rapid7_vulndb/tests/search_db.json

```diff
@@ -4,10 +4,10 @@
         "connection": {},
```

Comment:
Generate test file for `get_content`

Reply:
I think that we still need a test for `get_content`